### DISCUSSION TITLE
GPU assisted plotting? as CPU gets hot during plotting

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#2305] [Q&A]
Is there a way to utilize GPU for plotting? The CPU get burning hot and it can burn out fast making investing in farming less desirable?

when CPU is close to 100% load, would adding more plotting bog down existing plotting?

**Comment by hoffmang9** [comment]
GPUs are not conducive to plotting Chia proofs of space.
